### PR TITLE
[DROOLS-3883] Add running indicator

### PR DESCRIPTION
**Pull request by danielezonca**
https://issues.jboss.org/browse/DROOLS-3883

NOTE: the indicator was already removed in case of exception by `HasBusyIndicatorDefaultErrorCallback` 

@kkufova @Rikkola @gitgabrio

### CODE REVIEW

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties [hunk 1]
```diff
@@ -130,5 +130,6 @@ settings=Settings
 uploadWarning=Make sure the headers in the selected CSV file have not been modified. Otherwise, the spreadsheet may not be successfully imported.
 importFailedMessage=A CSV file with an incorrect header layout has been imported. Make sure the imported CSV file has unmodified headers.
 importErrorTitle=Error during import
+running=Running
```
Comment (kkufova) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties]: Please change this to `Running...` to be consistent with other loading indicators.

Comment (danielezonca) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties]: Done

Comment (kkufova) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties]: Thanks!

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/ScenarioSimulationEditorPresenter.java [hunk 2]
```diff
@@ -351,6 +352,7 @@ protected boolean isUberfireDocksInteractionEventToManage(UberfireDocksInteracti
     }
 
     protected void refreshModelContent(TestRunResult testRunResult) {
+        view.hideBusyIndicator();
```
Comment (gitgabrio) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/ScenarioSimulationEditorPresenter.java]: @danielezonca 
Please use this one
**CustomBusyPopup**  (see comment above)

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/ScenarioSimulationEditorPresenter.java [hunk 3]
```diff
@@ -298,6 +298,7 @@ public void onRunScenario(List<Integer> indexOfScenarioToRun) {
                         simulation::getScenarioByIndex
                 )
         );
+        view.showBusyIndicator(ScenarioSimulationEditorConstants.INSTANCE.running());
```
Comment (gitgabrio) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/ScenarioSimulationEditorPresenter.java]: @danielezonca 
Please use this one
**CustomBusyPopup**  (see firs line inside _loadContent_ method) that has been implemented specifically for ScenarioSimulation

**Comment by danielezonca** [comment]
jenkins retest this please

**Comment by kkufova** [comment]
@manstis, could you please merge this?

**Pull request merged**